benchttp/benchttp.go

```diff
@@ -0,0 +1,27 @@
+package benchttp
+
+// Report represents the result of a Benchttp benchmark run.
+type Report struct {
+	Benchmark Benchmark
+}
+
+// Benchmark represents the detailed collection of requests
+// done during a Benchttp benchmark run.
+type Benchmark struct {
+	Records []Record
+}
+
+// Times returns the recorded exectution time of the requests
+// as a slice of floats.
+func (b Benchmark) Times() []float64 {
+	s := make([]float64, len(b.Records))
+	for i, v := range b.Records {
+		s[i] = float64(v.Time)
```

Comment:
Interesting conversion 🤔

Reply:
```suggestion
		s[i] = float64(float64(float64(v.Time)))
```
(just to be sure)

Reply:
Woops, `Time` is still time in my mind ahah ! Good catch :)